package.json

```diff
@@ -2,14 +2,20 @@
   "name": "h5wasm",
   "version": "0.6.10",
   "description": "A high-level library for reading and writing HDF5 files from Javascript, using a wasm-compiled version of the HDF5 C library",
-  "types": "./src/hdf5_hl.d.ts",
   "type": "module",
-  "main": "./dist/node/hdf5_hl.js",
+  "main": "./dist/iife/hdf5_hl.js",
   "module": "./dist/esm/hdf5_hl.js",
+  "browser": "./dist/iife/h5wasm.js",
+  "types": "./src/hdf5_hl.d.ts",
   "exports": {
-    "types": "./src/hdf5_hl.d.ts",
-    "node": "./dist/node/hdf5_hl.js",
-    "import": "./dist/esm/hdf5_hl.js"
+    ".": {
+      "types": "./src/hdf5_hl.d.ts",
+      "import": "./dist/esm/hdf5_hl.js"
```

Comment:
Might be worth renaming the `esm` folder to `browser` or something like that. It's `esm` that made me think that the `node` folder contained a CJS build.

Reply:
I don't really want to rename the `esm` folder because that would break the current CDN paths e.g. https://cdn.jsdelivr.net/npm/h5wasm@latest/dist/esm/hdf5_hl.js

I'm happy to rename the `node` folder to `node_esm` if that would help...

Reply:
Hmm fair enough! No worries, it's fine as is.